### PR TITLE
Willys/consistency-standardization

### PR DESCRIPTION
**Pull request by wpschaub**
Introducing the need for and difference of consistency and standardization.

### CODE REVIEW

**Comment by wpschaub** [comment]
A short article discussing the need for and difference in consistency and standardization. Part of a rolling series of quick articles to cover goals of our common engineering vision.

File: content/consistency-standardization.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+# Back to the question ... why do we promote consistency and standardization?
+
+Why we love consistency and standardization:
+
+- **Consistency improves adoption** - Once we become familiar with a process or product, resistance to explore and embrace others will evaporate. For example, if you know how to use Office 365 Word, you can navigate yourself around Office 365 Excel or PowerPoint with familiarity and without friction.
+
+- **Consistency reduces cost and time** - Usability and learnability improves with consistency, which empowers engineering to deliver value quicker and continuously. Reducing time and anomalies, reduces cost - a favourable side-effect for investors and researchers.
+
+- **Consistency empowers automation** - We frequently say that "_if you do something twice or trice, consider automation!_" While this is solid guidance, automation becomes a lot easier if you have a consistent process and products at your disposal.
```
Comment (ShayVannery) [inline content/consistency-standardization.md]: Sorry, being nit-picky here: "Thrice" is misspelled.

**Review by ShayVannery — DISMISSED** [review]

File: content/consistency-standardization.md (hunk 1 above)
Comment (wpschaub) [reply content/consistency-standardization.md]: I love nit-picky and pedantic reviews :) Fixed.

**Review by RedPointMS — DISMISSED** [review]
I am generally in agreement. But I think it might be good idea to have another post on subject where standardization could be a 'slippery slope'. There are always to sides to this.

**Review by erwsch1 — DISMISSED** [review]

**Review by ShayVannery — APPROVED** [review]

**Comment by wpschaub** [comment]
@AndreKaminski any objections to this article, which is planned to be published tomorrow (Friday)? Worst case I can swap the branching/PR article with this one.

**Review by erwsch1 — APPROVED** [review]

**Review by AndreKaminski — APPROVED** [review]

**Pull request merged**